src/lib/package-generator.ts

```diff
@@ -26,7 +26,7 @@ export async function generatePackage(typing: TypingsData, availableTypes: Types
 	];
 	outputs.push(...typing.files.map(async file => {

```

Comment:
Need to rearrange this code a bit because `package.json` will be in the `files` list (so we copy it on line 31) and we'll end up with either the partial or the final file in the output folder depending on who wins the race.